extensions/wikia/Search/classes/IndexService/CrossWikiCore.php

```diff
@@ -65,113 +69,130 @@ protected function getWikiBasics() {
 		$response['all_domains_mv_wd'] = $response['domains_txt'];
 
 		$response['wiki_pagetitle_txt'] = str_replace( '$1 - ', '', ( \wfMessage( 'Pagetitle' )->text() ) );
+
 		return $response;
 	}
-	
+
 	/**
 	 * Retrieves the views on a weekly and monthly basis for this wiki
+	 *
 	 * @return array
 	 */
 	protected function getWikiViews() {
 		$result = [];
-		if ( $wvResponse = (new WikiViews)->getStubbedWikiResponse() ) {
-			$result = [ 'views_weekly_i' => $wvResponse['contents']['wikiviews_weekly']['set'], 'views_monthly_i' => $wvResponse['contents']['wikiviews_monthly']['set'] ];
+		if ( $wvResponse = ( new WikiViews )->getStubbedWikiResponse() ) {
+			$result = [
+				'views_weekly_i' => $wvResponse['contents']['wikiviews_weekly']['set'],
+				'views_monthly_i' => $wvResponse['contents']['wikiviews_monthly']['set']
+			];
 		}
+
 		return $result;
 	}
-	
+
 	/**
 	 * Retrieves WAM score for this wiki
+	 *
 	 * @return array
 	 */
 	protected function getWam() {
 		$response = [];
-		if ( $wamResp = (new Wam)->getStubbedWikiResponse() ) {
+		if ( $wamResp = ( new Wam )->getStubbedWikiResponse() ) {
 			$response['wam_i'] = $wamResp['contents']['wam']['set'];
 		}
+
 		return $response;
 	}
-	
+
 	/**
 	 * Retrieves stats like number of videos, number of images, number of articles, etc.
+	 *
 	 * @return array
 	 */
 	protected function getWikiStats() {
 		$service = $this->getService();
 		$data = $service->getApiStatsForWiki();
 		$response = [];
-		if ( (! empty( $data['query'] ) ) && ( isset( $data['query']['statistics'] ) ) ) {
+		if ( ( !empty( $data['query'] ) ) && ( isset( $data['query']['statistics'] ) ) ) {
 			foreach ( $data['query']['statistics'] as $key => $val ) {
 				$response[$key . '_i'] = $val;
 			}
 		}
-		$response['videos_i'] = (new WikiService)->getTotalVideos( $this->getWikiId() );
+		$response['videos_i'] = ( new WikiService )->getTotalVideos( $this->getWikiId() );
+
 		return $response;
 	}
-	
+
 	/**
 	 * Retrieves description, headline, etc.
+	 *
 	 * @return array
 	 */
 	protected function getVisualizationInfo() {
 		$response = [];
 		$service = $this->getService();
 
-		$message = $service->getSimpleMessage( 'wikiasearch2-crosswiki-description', array( $service->getGlobal( 'Sitename' ) ) );
+		$message =
+			$service->getSimpleMessage( 'wikiasearch2-crosswiki-description', [ $service->getGlobal( 'Sitename' ) ] );
 		$ds = Utilities::field( 'description' );
 		$ds = $ds == 'description' ? 'description_txt' : $ds;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "description_txt" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)